lib/rouge/lexers/supercollider.rb

```diff
@@ -122,7 +108,6 @@ def self.constants
 
         # treat curry argument as a special operator
         rule /_/, Name::Builtin
```

Comment:
This likely needs `\b` boundaries around it.

Reply:
Ooh, yep!